packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -491,4 +546,25 @@ public class AuthCognito : FlutterPlugin, ActivityAware, MethodCallHandler, Plug
       flutterResult.success(session.toValueMap());
     }
   }
+
+  fun prepareUpdateUserAttributeResult(@NonNull flutterResult: Result, @NonNull result: AuthUpdateAttributeResult) {
+    var updateUserAttributeResult = FlutterUpdateUserAttributeResult(result);
+    Handler (Looper.getMainLooper()).post {
+      flutterResult.success(updateUserAttributeResult.toValueMap());
+    }
+  }
+
+  fun prepareConfirmUserAttributeResult(@NonNull flutterResult: Result) {
+    var parsedResult = mutableMapOf<String, Any>();
+    Handler (Looper.getMainLooper()).post {
+      flutterResult.success(parsedResult);
+    }
+  }
+
+  fun prepareResendUserAttributeConfirmationCodeResult(@NonNull flutterResult: Result, @NonNull result: AuthCodeDeliveryDetails) {
```

Comment:
It's kind of odd that `resendSignUpCode` doesn't return `AuthCodeDeliveryDetails` as well, but I guess that's the case.